ui/app/components/secure-variable-form.js

```diff
@@ -33,7 +33,10 @@ export default class SecureVariableFormComponent extends Component {
   @tracked duplicatePathWarning = null;
 
   get shouldDisableSave() {
-    return !!this.JSONError || !this.args.model?.path;
+    const disallowedPath =
+      this.args.model?.path.startsWith('nomad/') &&
+      !this.args.model?.path.startsWith('nomad/jobs/');
```

Comment:
`thought` `non-blocking`: are we using the correct variable naming here? I get that its disallowed, but why?